packages/modal/tests/integration/components/nucleus-modal-test.js

```diff
@@ -138,4 +139,65 @@ module('Integration | Component | nucleus-modal', function(hooks) {
       assert.ok(true, 'no a11y errors found!');
     });
   });
+
+  //Note: All Visual Regression Tests for modal based objects require a div container with a fixed height
+  test('visual regression for modal', async function(assert) {
+    await render(hbs`<div id="viewport-container" style="height:900px">
```

Comment:
Can you check if it's possible to get rid of this implementation? 

1. Tweak the onReady.js script to achieve similar results?
2. https://github.com/garris/BackstopJS/issues/502#issuecomment-324069356
3. https://github.com/garris/BackstopJS#trigger-screen-capture-via-selector

Reply:
Fixed in the latest commit.